xmlseq.go

```diff
@@ -36,54 +36,62 @@ var NO_ROOT = NoRoot // maintain backwards compatibility
 // as map["#seq"]<int value>.
 // If the optional argument 'cast' is 'true', then values will be converted to boolean or float64 if possible.
 // NOTE: "#seq" key/value pairs are removed on encoding with msv.Xml() / msv.XmlIndent().
-//	• attributes are a map - map["#attr"]map["attr_key"]map[string]interface{}{"#text":<aval>, "#seq":<num>}
-//	• all simple elements are decoded as map["#text"]interface{} with a "#seq" k:v pair, as well.
-//	• lists always decode as map["list_tag"][]map[string]interface{} where the array elements are maps that
-//	  include a "#seq" k:v pair based on sequence they are decoded.  Thus, XML like:
-//	      <doc>
-//	         <ltag>value 1</ltag>
-//	         <newtag>value 2</newtag>
-//	         <ltag>value 3</ltag>
-//	      </doc>
-//	  is decoded as:
-//	    doc :
-//	      ltag :[[]interface{}]
-//	        [item: 0]
-//	          #seq :[int] 0
-//	          #text :[string] value 1
-//	        [item: 1]
-//	          #seq :[int] 2
-//	          #text :[string] value 3
-//	      newtag :
-//	        #seq :[int] 1
-//	        #text :[string] value 2
-//	  It will encode in proper sequence even though the MapSeq representation merges all "ltag" elements in an array.
-//	• comments - "<!--comment-->" -  are decoded as map["#comment"]map["#text"]"cmnt_text" with a "#seq" k:v pair.
-//	• directives - "<!text>" - are decoded as map["#directive"]map[#text"]"directive_text" with a "#seq" k:v pair.
-//	• process instructions  - "<?instr?>" - are decoded as map["#procinst"]interface{} where the #procinst value
-//	  is of map[string]interface{} type with the following keys: #target, #inst, and #seq.
-//	• comments, directives, and procinsts that are NOT part of a document with a root key will be returned as
-//	  map[string]interface{} and the error value 'NoRoot'.
-//	• note: "<![CDATA[" syntax is lost in xml.Decode parser - and is not handled here, either.
-//	   and: "\r\n" is converted to "\n"
 //
-//	NOTES:
```

Comment:
Sorry I'm using VSCode and the formatter used by `gopls` thinks this is the right way to format these comments. Let me know if you want them reverted.